src/main/java/uk/gov/hmcts/reform/sscs/model/SessionCaseCodeMapping.java

```diff
@@ -0,0 +1,47 @@
+package uk.gov.hmcts.reform.sscs.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Value;
+
+@Value
+@Builder(toBuilder = true)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class SessionCaseCodeMapping {
```

Comment:
benefitCode, benefitDescription and ccdKey can be abstracted out, benefitCode and benefitDescription are duplicated throughout and can be replaced with a reference, (maybe the enum benefit?).
ccdKey is the combination of benefitCode  and issuecode